setup.cfg

```diff
@@ -1,3 +1,3 @@
 [flake8]
-ignore = E501,E226,E731,F405
+ignore = E501,E226,E731,F405,F403,F401
```

Comment:
Ignoring F403 is okay, but please don't ignore F401, clean up unused imports instead.